<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectDataAnalysis</class>
 <widget class="QMainWindow" name="IndirectDataAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>764</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Indirect Data Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="twIDATabs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabElwin">
       <attribute name="title">
        <string>Elwin</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabMSD">
       <attribute name="title">
        <string>MSD Fit</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabIqt">
       <attribute name="title">
        <string>I(Q, t)</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabIqtFit">
       <attribute name="title">
        <string>I(Q, t) Fit</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabConvFit">
       <attribute name="title">
        <string>ConvFit</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabJumpFit">
       <attribute name="title">
        <string>F(Q) Fit</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_bottom">
      <item>
       <widget class="QPushButton" name="pbHelp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Open interface help page in your web browser.</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPythonExport">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Export a Python script for the algorithms run on this tab.</string>
        </property>
        <property name="text">
         <string>Py</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbRun">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbManageDirs">
        <property name="text">
         <string>Manage Directories</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>twIDATabs</tabstop>
  <tabstop>pbHelp</tabstop>
  <tabstop>pbRun</tabstop>
  <tabstop>pbManageDirs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
